os/kernel/sched/sched_releasetcb.c

```diff
@@ -150,40 +150,38 @@ int sched_releasetcb(FAR struct tcb_s *tcb, uint8_t ttype)
 	int ret = OK;
 
 	if (tcb) {
+		/* Release the some of task's resources if PID was assigned.
+		 * PID zero is reserved for the IDLE task.  The TCB of the IDLE
+		 * task is never release so a value of zero simply means that
+		 * the process ID was never allocated to this TCB.
+		 */
+		if (tcb->pid) {
```

Comment:
'tcb->pid != 0' is same as below code but Consider above comment, it is better to make it explicitly.

Reply:
Yes, but I will leave it as it is for code unity. Thanks.